rs/bitcoin/ckbtc/minter/ckbtc_minter.did

```diff
@@ -321,6 +319,14 @@ type ReimbursementReason = variant {
     };
 };
 
+type DiscardedReason = variant {
```

Comment:
I think "discarded" is the wrong word. It would be more accurate to say that we "retain UTXOs" (i.e., we keep them but perform no further action), so here we could say `RetentionReason`.

Reply:
Thanks for the suggestion, makes sense! At the end I went for `suspended` because `retain` seems a bit too neutral to me (`retain` is part of the standard library, e.g. for [VecDeque](https://doc.rust-lang.org/std/collections/struct.VecDeque.html#method.retain)). PTAL